src/utils/form-validation.ts

```diff
@@ -45,3 +45,12 @@ export const validateIsContactNameValid = ({ name, id }: Omit<Contact, 'address'
     ? i18n.t('A contact with this name already exists') + `: ${existingContact.address}`
     : true
 }
+
+export const validateIsWalletNameValid = ({ name, previousName }: { name: string; previousName?: string }) =>
```

Comment:
Nit: I didn't mention it last time, but why prefixing the name with `validate`? It's a bit redundant to "validate if valid"?

Why not just `isWalletNameValid`? :)

Reply:
haha very good point! I did it because I wanted to use the `isAddressValid` function inside a validation function and I could not have 2 functions with the same name 😅 

```ts
export const validateIsAddressValid = (value: string) => isAddressValid(value) || i18n.t('This address is not valid')
```

How do you propose I address (pun intended) this?

Reply:
I can do this!

```ts
import { isAddressValid as _isAddressValid } from '@alephium/sdk'
```

Reply:
Or `isAddressHashValid`? :)